ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1202,3 +1202,14 @@ literals:
 - thermal_energy_accumulator
 - ground_isolation_command
 - ground_temperature_sensor
+
+## New fields for Leak Detection System
+- water_leak_cable_distance_sensor
+
+## New fields for Freezer (Walk in Cold room)
+- defrost_run_status
```

Comment:
since these aren't needed for this PR, please remove them. they are covered in another PR.